CommandTests/Generated/list.0.a330395c.md

```diff
@@ -8,17 +8,19 @@
 
 # Expected output
 ```
-- FILE_REFERENCES
-- BUILD_PHASES
-- COPY_FILES
-- TARGETS
-- HEADERS
-- SOURCES
-- RESOURCES
-- CONFIGURATIONS
-- SETTINGS
-- RESOLVED_SETTINGS
-- SOURCE_TREES
-- LINKED_DEPENDENCIES
+  COMPARTOR TAG                | DEFAULT  
```

Comment:
I wonder if the wording needs a slight tweak or possibly additional text is needed?

e.g. It may not be obvious what Default means in this context?


Some ideas:

```
  COMPARTOR TAG                | INCLUDED BY DEFAULT  
------------------------------------------------------
- FILE_REFERENCES              | Yes
- BUILD_PHASES                 | Yes
- COPY_FILES                   | Yes
- TARGETS                      | Yes
- HEADERS                      | Yes
- SOURCES                      | Yes
- RESOURCES                    | Yes
- CONFIGURATIONS               | Yes
- SETTINGS                     | Yes
- RESOLVED_SETTINGS            | No
- SOURCE_TREES                 | Yes
- LINKED_DEPENDENCIES          | Yes
```

It's a bit too long

```
The following list shows all available comparator tags along with their default inclusion status when tags aren't explicitly specified.

  COMPARTOR TAG                | INCLUDED 
------------------------------------------------------
- FILE_REFERENCES              | Yes
- BUILD_PHASES                 | Yes
- COPY_FILES                   | Yes
- TARGETS                      | Yes
- HEADERS                      | Yes
- SOURCES                      | Yes
- RESOURCES                    | Yes
- CONFIGURATIONS               | Yes
- SETTINGS                     | Yes
- RESOLVED_SETTINGS            | No
- SOURCE_TREES                 | Yes
- LINKED_DEPENDENCIES          | Yes
```

Reply:
Thanks @kwridan, good idea. I went for the 2nd option